BuildMaster/ci-cd/continuous-integration/importing-builds/#.md

```diff
@@ -4,31 +4,29 @@ sequence: 300
 show-headings-in-nav: true
 ---
 
-BuildMaster is designed to continuously deliver your applications and components from source code to production, but unlike monolith CI/CD platforms like Azure DevOps or GitLab, you can work the tools and processes you're already using, including dedicated Continuous Integration  servers like Jenkins and TeamCity.
+BuildMaster can import build artifacts from any of three internal sources: 1) CI servers; 2) Package servers (such as ProGet); 3) Drop folders. In addition, BuildMaster can use OtterScript, a programmatic language, to fully customise build imports using build triggers.
 
-While BuildMaster can certainly perform [continuous integration and build automation](/docs/buildmaster/builds/continuous-integration), there are a lot of good reasons to continue use your existing continuous integration servers:
+Collectively, this functionality solves the problems created by build processes that unfold as a series of 'siloed' (i.e.: disconnected) processes and tools.
 
-- **Save time and resources on retraining**: if your developers are already familiar with the CI tools and processes you've setup, they won't have to spend time learning something new
-- **Reuse complex automations**: processes like automated testing are complex to develop and are often fragile. Moving these processes to a new system like BuildMaster will require some degree of effort that might be better spent elsewhere
-- **Keep your existing infrastructure**: you can use the CI server infrastructure that you've already built and is providing you with value
+BuildMaster takes care of automation gaps by using OtterScript to model your existing process/tool stack instead of rewriting everything to fit a monolith tool. It also gives you visibility into the whole process so that each of these automations looks like environments in a pipeline.
 
-A large percentage of customers use BuildMaster in exactly this manner.
+Fulfilling Automation Directives & Avoiding the Monolith Trap
 
-:::attention {.best-practice}  
-If you want to keep using the CI tool you already love and use, BuildMaster lets you do it. But BuildMaster is as solid a Continuous Integration server as a Continuous Delivery tool. Whether you build inside the program or continue using an outside tool, you can do it with BuildMaster.
-:::
+Organisations must often manage hundreds of applications spread across Jenkins, TeamCity, etc. inside their portfolio—many of which have dedicated, legacy build processes that function using drop folders. 'Build pipelines' are actually chains of disconnected, output sequential tasks like:
+
+Pulling artifacts from source control into Jenkins
```

Comment:
Pulling code from source repsository and building in jenkins and saving as an artifact (but a shorter version than this)

Jenkins turnst "source code into artifacts"